assets/packages/expressions/em_javascript.js

```diff
@@ -1233,7 +1233,10 @@ function LEMval(alias)
                     return "";
                 }
 
-                var checkNumericRegex = new RegExp(/^(-)?[0-9]*(,|\.)[0-9]*$/);
+                var isNumericRegex = new RegExp(/^(-)?[0-9]*(\.)[0-9]*$/);
```

Comment:
Same as in the other branch. Maybe create a function? Up to you :) Let me know. thanks!

Reply:
Remove unrelated fix :)